app/components/meeting/edit-meeting-modal.js

```diff
@@ -231,26 +231,44 @@ export default class MeetingEditMeetingComponent extends Component {
     this.args.meeting.numberRepresentation = this.numberRepresentation;
     this.args.meeting.mainMeeting = this.selectedMainMeeting;
 
-    if (this.enableDigitalAgenda) {
-      const currentMeetingSecretary = yield this.args.meeting.secretary;
-      if (currentMeetingSecretary?.uri !== this.secretary.uri) {
-        this.args.meeting.secretary = this.secretary;
-        const decisionActivities = yield this.store.queryAll('decision-activity', {
-          'filter[treatment][agendaitems][agenda][created-for][:id:]':
-            this.args.meeting.id,
+    if (this.args.meeting.secretary != this.secretary) {
+      const minutes = yield this.args.meeting.minutes;
+      if (minutes) {
+        const latestMinutesPiecePart = yield this.store.queryOne('piece-part', {
+          filter: {
+            minutes: { ':id:': minutes.id },
+            ':has-no:next-piece-part': true,
+          },
         });
-        for (let decisionActivity of decisionActivities.slice()) {
-          decisionActivity.secretary = this.secretary;
-          yield decisionActivity.save();
-          if (this.enableDigitalAgenda) {
-            const report = yield this.store.queryOne('report', {
-              'filter[:has-no:next-piece]': true,
-              'filter[decision-activity][:id:]': decisionActivity.id,
-            });
-            const pieceParts = yield report?.pieceParts;
-            if (pieceParts?.length) {
-              yield this.decisionReportGeneration.generateReplacementReport.perform(report);
-            }
+        const currentSecretary = yield this.args.meeting.secretary
+        const currentSecretaryPerson = yield currentSecretary.person;
+        const currentSecretaryPersonFullName = yield currentSecretaryPerson.get('fullName');
+        const newSecretaryPerson = yield this.secretary.person;
+        const newSecretaryPersonFullName = yield newSecretaryPerson.get('fullName');
+        latestMinutesPiecePart.value = latestMinutesPiecePart.value.replace(currentSecretaryPersonFullName, newSecretaryPersonFullName);
+        yield latestMinutesPiecePart.save();
+
+        const fileMeta = yield this.exportPdf.perform(minutes);
+        if (fileMeta) {
+          yield this.replaceMinutesFile(minutes, fileMeta.id);
+        }
+      }
```

Comment:
**chore**: undo these changes

The feature to re-generate the minutes when changing the secretary was already added in https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1887, let's remove the changes here so we don't implement the same thing twice.